apps/console/src/features/claims/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -39,14 +40,27 @@ import {
     Message
 } from "@wso2is/react-components";
 import Axios from "axios";
-import React, { FunctionComponent, ReactElement, useEffect, useMemo, useRef, useState } from "react";
+import { IdentityAppsError } from "modules/core/dist/types/errors/identity-apps-error";
```

Comment:
Shall we import it from `@wso2is/core`?

Reply:
Addressed by 309ce1299dddd6673c8be1280c6d4230a3b2a27b.